stable/enterprise/values.yaml

```diff
@@ -1011,8 +1111,15 @@ reports:
   extraEnv: []
 
   ## @param reports.resources Resource requests and limits for Anchore Reports pods
+  ## ref: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+  ## Commented values below are just a suggested baseline. Contact Anchore support for deployment specific recommendations.
```

Comment:
It may be worth putting a note here pointing to the `use_volume` config option as that will impact memory requirements. Otherwise they'll need to set higher memory limits here if they have large amounts of data.